lib/generators/allinson_flex/templates/config/metadata_profile/default_schema.json

```diff
@@ -422,6 +422,14 @@
               }
             ]
           },
+          "multi_value": {
+            "type": "boolean",
+            "description": "Set whether the property is singular or multiple",
+            "default": "true",
+            "examples": [
+              true
+            ]
+          },
```

Comment:
this is just how I imagine things will go. but....we can decide that this defaults to false instead too.